ground/gcs/src/plugins/config/outputchannelform.cpp

```diff
@@ -86,14 +86,6 @@ OutputChannelForm::OutputChannelForm(const int index, QWidget *parent, const boo
     ui.actuatorLink->setChecked(false);
     connect(ui.actuatorLink, SIGNAL(toggled(bool)), this, SLOT(linkToggled(bool)));
 
-    // Trigger when autopilot is connected
-    connect(this, SIGNAL(autoPilotConnected()), this, SLOT(onAutopilotConnect()));
```

Comment:
why was this removed? isn't it calling a method on the parent that refreshes the initial values (e.g. neutral?)